src/plugins/chart_expressions/expression_pie/public/components/chart_split.tsx

```diff
@@ -7,9 +7,9 @@
  */
 
 import React from 'react';
-import { Accessor, AccessorFn, GroupBy, GroupBySort, SmallMultiples } from '@elastic/charts';
+import { Accessor, AccessorFn, GroupBy, SmallMultiples } from '@elastic/charts';
+import { Predicate } from '@elastic/charts/dist/common/predicate';
```

Comment:
It looks like `@elastic/charts` does not export directly the `Predicate` type. Maybe this should be fixed upstream?

Reply:
You are right, sorry about that. Let me export it and I can fix on a subsequent PR ok?

Reply:
sure